code/models/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require(__dirname + '/../config/config.json')[env];
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable], config);
+} else {
+  sequelize = new Sequelize(config.database, config.username, config.password, config);
+}
+
+fs
+  .readdirSync(__dirname)
```

Comment:
So `.readdirSync` grabs the directory we tell it to and it returns an array of the files inside. We `filter` through and only take the ones that have `.js` and to each of them we add a 'model' which is just a path that adds to our `db` object a new key of that filename and the `model` import of our connected Sequelized table/data object.